src/version.h

```diff
@@ -35,8 +35,11 @@ static const int PROTOCOL_VERSION = 70002;
 // intial proto version, to be increased after version/verack negotiation
 static const int INIT_PROTO_VERSION = 209;
 
+// In this version, 'getheaders' was introduced.
+static const int GETHEADERS_VERSION = 31800;
```

Comment:
This should be set to 70002 (3.2 where we introduced 'getheaders') and NOBLKS_VERSION_END should be set to this so we only ask for blocks from 3.2 nodes or greater